dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleMetadataConnector.java

```diff
@@ -125,6 +125,9 @@ public void addTasksTo(@Nonnull List<? super Task<?>> out, @Nonnull ConnectorArg
     String ownerInList = toInList(arguments.getSchemata());
     String whereCondOwner = ownerInList == null ? "" : " WHERE OWNER IN " + ownerInList;
     String whereCondTableOwner = ownerInList == null ? "" : " WHERE TABLE_OWNER IN " + ownerInList;
+    String whereCondSequenceNotSystemGeneratedAndOwner =
+        ownerInList == null ? " WHERE SEQUENCE_NAME NOT LIKE 'ISEQ$$_%'"
+            : " WHERE SEQUENCE_NAME NOT LIKE 'ISEQ$$_%' AND SEQUENCE_OWNER IN " + ownerInList;
```

Comment:
Nit: note that `_` has a special meaning in `LIKE` operator patterns. Consider using the `ESCAPE` clause.

Reply:
Added `ESCAPE '\'`, but escaped in a Java string literal (no \p\u\n intended)